src/Experiences/Experiences.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Experiences.css';
+
+export const Experiences = ({ strain }) => {
+  return (
+    <section>
+      
```

Comment:
I find it helpful to throw some temp placeholder text in empty components that actually render - so y ou know they're rendering properly

Reply:
Im in the same boat here^

Reply:
Good call! I think I just got caught second-guessing what I wanted to do with Experiences. My thoughts are to render all the saved strains, and potentially use local storage.